packages/webapp/pages/onboarding.tsx

```diff
@@ -510,7 +521,11 @@ export function OnboardPage(): ReactElement {
             {activeScreen === OnboardingStep.ContentTypes && <ContentTypes />}
             {activeScreen === OnboardingStep.Plus && (
               <PaymentContextProvider>
-                <OnboardingPlusStep onClickNext={onClickNext} />
+                {isPlusCheckout ? (
```

Comment:
Do we want to utilize `isPlusAvailable` from context to not show this step for people from sanctioned countries?

Reply:
Right. Actually, we should probably not show the whole Plus page regardless of experiment enrollment if they are within the sanctioned countries.

Reply:
I will do it in a separate PR to not block this and carefully test it 🙏